website/public/index.ejs.html

```diff
@@ -432,6 +452,7 @@ <h2 id="linter-feature-comparison">Features</h2>
               <td class="bad"><span aria-label="no">❌</span></td>
               <td class="good"><span aria-label="yes">✅</span></td>
               <td class="bad"><span aria-label="no">❌</span></td>
+              <td class="bad"><span aria-label="no">❌</span></td>
```

Comment:
Blocking: SonarLint's website claims it has an IntelliJ (aka WebStorm) plugin. Does it not work for JS code? https://www.sonarlint.org/intellij/